src/OpenTelemetry.Exporter.Zipkin/ZipkinExporter.cs

```diff
@@ -35,8 +38,18 @@ public ZipkinExporter(ZipkinExporterOptions options, HttpClient client = null)
         Guard.ThrowIfNull(options);
 
         this.options = options;
-        this.maxPayloadSizeInBytes = (!options.MaxPayloadSizeInBytes.HasValue || options.MaxPayloadSizeInBytes <= 0) ? ZipkinExporterOptions.DefaultMaxPayloadSizeInBytes : options.MaxPayloadSizeInBytes.Value;
+        this.maxPayloadSizeInBytes = (!options.MaxPayloadSizeInBytes.HasValue || options.MaxPayloadSizeInBytes <= 0)
+            ? ZipkinExporterOptions.DefaultMaxPayloadSizeInBytes
+            : options.MaxPayloadSizeInBytes.Value;
         this.httpClient = client ?? options.HttpClientFactory?.Invoke() ?? throw new InvalidOperationException("ZipkinExporter was missing HttpClientFactory or it returned null.");
+
+#if NET
+        // See: https://github.com/dotnet/runtime/blob/280f2a0c60ce0378b8db49adc0eecc463d00fe5d/src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.cs#L767
+        this.synchronousSendSupportedByCurrentPlatform = !OperatingSystem.IsAndroid()
+            && !OperatingSystem.IsIOS()
+            && !OperatingSystem.IsTvOS()
+            && !OperatingSystem.IsBrowser();
```

Comment:
How do we test these?

Reply:
I'm going to merge this let's tackle tests or example app as a separate thing. Because I want to make sure we do something sensible here. When I did this work initally on contrib I had to get a MAUI app up and running to test things. Couldn't figure out how to do unit tests or CI (what we want more than an example app). In the end I didn't check in the MAUI part because it would have forced everyone to install android sdks into their workloads just to build the solution. Seemed way too heavy handed for what seems to be a fringe thing at the moment (running SDK on mobile).

Reply:
> I'm going to merge this let's tackle tests or example app as a separate thing.

+1, @CodeBlanch could you create a tracking issue?

Reply:
#5831